src/components/ec-btn-dropdown/ec-btn-dropdown.story.js

```diff
@@ -0,0 +1,41 @@
+import EcBtnDropdown from './ec-btn-dropdown.vue';
+
+export default {
+  title: 'ButtonDropdown',
+  component: EcBtnDropdown,
+};
+
+const Template = (args, { argTypes }) => ({
+  props: Object.keys(argTypes),
+  components: {
+    EcBtnDropdown,
+  },
+  template: `
+  <div class="tw-m-24 tw-p-24 tw-flex tw-justify-center tw-bg-gray-0">
+    <ec-btn-dropdown :itemsDropdown="items" :isDisabled="isDisabled" :textButton="textButton" />
+  </div>
+  `,
+});
+
+export const basic = Template.bind({});
```

Comment:
We should have the button fully working in this story. Without events for click and change attached, the button doesn't do anything.